traffic_server/plugins/sst/sst.lua

```diff
@@ -0,0 +1,99 @@
+--  Licensed to the Apache Software Foundation (ASF) under one
+--  or more contributor license agreements.  See the NOTICE file
+--  distributed with this work for additional information
+--  regarding copyright ownership.  The ASF licenses this file
+--  to you under the Apache License, Version 2.0 (the
+--  "License"); you may not use this file except in compliance
+--  with the License.  You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+--  Unless required by applicable law or agreed to in writing, software
+--  distributed under the License is distributed on an "AS IS" BASIS,
+--  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--  See the License for the specific language governing permissions and
+--  limitations under the License.
+
+package.path = package.path .. ";/opt/trafficserver/script/uuid.lua"
+local uuid = require("uuid")
+local QUERY_STRING_PARAMS = ""
+
+function __init__(argtb)
+    local unique_id = tostring( {} ):sub(8) -- memory address of blank table
+    local x = unique_id + os.time() -- plus current time
```

Comment:
you can use ts.now() instead here 
https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/ts_lua.en.html#ts-now